src/file.js

```diff
@@ -24,22 +24,8 @@ Sk.builtin.file = function (name, mode, buffering) {
     } else if (this.name === "/dev/stderr") {
         this.fileno = 2;
     } else {
-        if (Sk.inBrowser) {  // todo:  Maybe provide a replaceable function for non-import files
```

Comment:
Maybe I'm missing some other piece, but I just built this into my skulpt for Runestone and I get the following error trying to run @eah13 's example code.

```
with open("foo.txt", "w") as f:
  print("Writing to foo.txt...\n")
  f.write("With open(...) now works on trinket!\n")
  f.write("yay for idiomatic Python!")
  
with open("foo.txt") as f:
  text = f.read()
  print("Contents of foo.txt:")
  print(text)
```
I added an element with the id="foo.txt"
The error I get is:  

```
ExternalError: File not found: 'foo.txt' on line 1
```

Even worse is that it appears to have broken all file processing that used to work.  For example:  http://interactivepython.org/runestone/static/thinkcspy/Files/Iteratingoverlinesinafile.html  works with a skulpt pre-this-PR but that same page fails with `ExternalError: File not found: 'qbdata.txt' on line 1` using this build.